Extension/package.json

```diff
@@ -282,2546 +282,2593 @@
-                  "when": {
-                    "type": "string",
-                    "pattern": "\\w*\\$\\(basename\\)\\w*",
-                    "default": "$(basename).ext",
-                    "markdownDescription": "%c_cpp.configuration.filesExcludeWhen.markdownDescription%"
+          "C_Cpp.default.browse.databaseFilename": {
+            "type": "string",
+            "markdownDescription": "%c_cpp.configuration.default.browse.databaseFilename.markdownDescription%",
+            "scope": "machine-overridable"
+          },
+          "C_Cpp.default.browse.limitSymbolsToIncludedHeaders": {
+            "type": "boolean",
+            "default": true,
+            "markdownDescription": "%c_cpp.configuration.default.browse.limitSymbolsToIncludedHeaders.markdownDescription%",
+            "scope": "resource"
+          },
+          "C_Cpp.default.systemIncludePath": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "uniqueItems": true,
+            "markdownDescription": "%c_cpp.configuration.default.systemIncludePath.markdownDescription%",
+            "scope": "machine-overridable"
+          },
+          "C_Cpp.default.customConfigurationVariables": {
+            "type": [
+              "object",
+              "null"
+            ],
+            "default": null,
+            "patternProperties": {
+              "(^.+$)": {
+                "type": "string"
+              }
+            },
+            "markdownDescription": "%c_cpp.configuration.default.customConfigurationVariables.markdownDescription%",
+            "scope": "machine-overridable"
+          },
+          "C_Cpp.default.enableConfigurationSquiggles": {
+            "type": "boolean",
+            "default": true,
+            "markdownDescription": "%c_cpp.configuration.default.enableConfigurationSquiggles.markdownDescription%",
+            "scope": "resource"
+          },
+          "C_Cpp.default.dotConfig": {
+            "type": "string",
+            "default": null,
+            "markdownDescription": "%c_cpp.configuration.default.dotConfig.markdownDescription%",
+            "scope": "resource"
+          }
+        }
+      },
+      {
+        "title": "Vcpkg",
+        "properties": {
+          "C_Cpp.vcpkg.enabled": {
+            "type": "boolean",
+            "default": true,
+            "markdownDescription": "%c_cpp.configuration.vcpkg.enabled.markdownDescription%",
+            "scope": "resource"
+          }
+        }
+      },
+      {
+        "title": "Debugger",
+        "properties": {
+          "C_Cpp.debugger.useBacktickCommandSubstitution": {
+            "type": "boolean",
+            "default": false,
+            "markdownDescription": "%c_cpp.configuration.debugger.useBacktickCommandSubstitution.markdownDescription%",
+            "scope": "window"
+          }
+        }
+      },
+      {
+        "properties": {
+          "C_Cpp.maxConcurrentThreads": {
```

Comment:
These "max" settings go in the "IntelliSense" section

Reply:
I disagree -- it applies to Code Analysis too. I'd say it belongs uncategorized. There's already a C_Cpp.references.maxConcurrentThreads and there's no C_Cpp.intelliSense.maxConcurrentThreads because that hasn't been implemented, but the closest equivalent is C_Cpp.intelliSense.maxCachedProcesses.